### DISCUSSION TITLE
programming language.

### DISCUSSION THREAD

**Discussion by ChaoticTechSupport** [Godot-Community-Games-Org/game1#2] [Polls]
Hi! what programming language should we use?

**Comment by GodotCommunityGames** [comment]
Oh man that's a good question. I didn't even think of that. I'm voting GDScript cuz that's what I've been using, and that's probably what most people use. But also once we get this community set up and start developing we could have different games going if there's enough interest in doing multiple games, but yeah each game we gotta decide which language or if both is acceptable, so good poll question!

**Comment by Yonodactyl** [comment]
I would say it depends. What do majority of the devs know? I'm familiar with C#, but write most of my games using GDScript

**Comment by AnonymousAggregator** [comment]
a preference for GDscript with a touch of C# where required.

**Comment by ghost** [comment]
I think it would make sense to stick to GDscript if these projects are intended as potential examples/jumping off points for the community.

**Reply by LightInn** [reply]
but then it could also be an exemple on "how to integrate a bit of C#" , but I agree with you with the fact it's should be gdscript focused if possible

**Reply by ghost** [reply]
Oh for sure that would be a very valuable example !
